src/Drupal/DrupalAutoloader.php

```diff
@@ -125,6 +125,8 @@ public function register(Container $container): void
                 'tokens',
                 'search_api',
                 'pathauto',
+                'translation',
+                'batch',
```

Comment:
Drupal core doesn't define this in system_hook_info. It looks like it's more `locale` module specific. We need to add dynamic load support for module_load_include